plan.md

```diff
@@ -1,9 +1,282 @@
+  + Date for MVP
+  + Number of _active_ people by the end of the calendar year (based on repeated interaction with the app, [not time spent on it](https://github.com/dwyl/hq/issues/149#issuecomment-269154087))
+
+#### What?
+
+#### First products
+> Our **first products** will be [time](github.com/dwyl/time) (name TBC!) and [tudo](github.com/dwyl/tudo).
+
+Built with a modular architecture, this application will be a cross between a
+to-do list and a time tracking application.
+
+Our products will benefit from a positive [network effect](https://medium.com/evergreen-business-weekly/the-power-of-network-effects-why-they-make-such-valuable-companies-and-how-to-harness-them-5d3fbc3659f8#.6w6z2hjew) where increased usage of the product leads to it being a more valuable product for everyone (plays into open source).
+
+#### Key aims:
++ Effortlessness (e.g. one-click timer start)
++ Minimum friction (e.g. If a GitHub URL is added as a task, all of the data is pulled from GitHub and filled in automatically)
++ Improved focussed time
++ Effective time management (understand where your time is going & how to improve)
++ Improved team communication
++ Focus on the individual's potential/productivity first
++ Ability to share with the team built in from the beginning so people know
+when they would be interrupting ("I'm on fire" one-click warning to others
+:wink:) - key is to do a lot of research around what people _want_ to share
+and what they want to be shared _with them_
+
+#### CRM
+We have outgrown our 'spreadsheet' customer relationship management (CRM) system
+and need a place where we can **securely and accessibly** store all our client
+communications so that the *whole team* can see and contribute to these.
+
+Our CRM *code* will be **fully open source**, following our [guiding principles](https://github.com/dwyl/phase-two#guiding-principles) and
+removing as many barriers to entry/use as possible for small businesses with a
+social impact. The data however, will not be accessible to anyone outside of the
+organisation or sold to 3rd parties (a la [LinkedIn](https://www.forbes.com/sites/georgeanders/2012/06/27/how-linkedin-strategy/#c4e2c1fdf82c)).
+
+Value lies in:
++ Linking all of the branches of our business together
++ Excellent low risk learning project for learning our new stack
++ Modular architecture which is easy to extend *without* becoming super patchy
+  + Modules re-used across multiple products and services projects
++ Transparency on business development for all of our team
++ Central location where data can easily be entered as our business development
+efforts expand with [Services](#services) and [Training](#training) offerings
++ 50% of all charitable organisations we have spoken to have told us that their
+current CRM is not fit for purpose, we could easily begin to provide a solution
+to this problem if we desired
+
+> There are many 'small' enhancements we will be making to increase the effectiveness
```

Comment:
To me, this explains our mindset with regards to approaching things we build. Does it now explain it adequately?